### ISSUE TITLE
Upgrade openTelemetry collector image in Multi cluster hack scripts 

### ISSUE THREAD

**Issue by josunect** [kiali/kiali#7293]
### Which dependency needs to be upgraded, and why?

The openTelemetry image would need to be upgraded to the k8s one, that would be the image by default: https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/UPGRADING.md#0880-to-0890
It was updated in this PR to use the contrib one, but the CI was failing, so it would probably require additional configuration.

### What is the current version of the dependency?

repository: "otel/opentelemetry-collector-contrib"

### Which version to upgrade to?

repository: "otel/opentelemetry-collector-k8s"

### Changelog link

Provide the dependency changelog link, if there is one available

https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/UPGRADING.md#0880-to-0890

**Comment by nrfox** [comment]
I think this is done now?

**Comment by josunect** [comment]
> I think this is done now?

I added the image repo value "otel/opentelemetry-collector-contrib" to the yaml, but probably we should considered to change to "otel/opentelemetry-collector-k8s"? As that is the one that will be used as a default. 
I try to use first the "otel/opentelemetry-collector-k8s", but the CI was still failing (Apparently they were no traces for what I saw in the screenshot), so, to fix it quickly I changed to "otel/opentelemetry-collector-contrib".

It can be closed if we consider it won't be necessary to use  "otel/opentelemetry-collector-k8s"
